controls/V-72841.rb

```diff
@@ -20,6 +20,12 @@
 uri: http://iase.disa.mil
 -----------------
 =end
+
+PG_VER = attribute(
+  'pg_version',
+  description: "The version of the PostgreSQL process which is being inspected (tested)",
+)
+
 PG_DBA = attribute(
   'pg_dba',
   description: 'The postgres DBA user to access the test database',
```

Comment:
We should always have a default value for attributes to ensure the profile runs in the 'no input' mode.